pkg/scheduler/framework/plugins/interpodaffinity/scoring_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 	plugintesting "k8s.io/kubernetes/pkg/scheduler/framework/plugins/testing"
 	"k8s.io/kubernetes/pkg/scheduler/internal/cache"
+	tf "k8s.io/kubernetes/pkg/scheduler/testing/framework"
```

Comment:
```suggestion
	testingframework "k8s.io/kubernetes/pkg/scheduler/testing/framework"
```

Reply:
`tf` is already used in many places, for example: https://github.com/kubernetes/kubernetes/blob/0c645922edcc06adff43c70c02fb56751364bbb5/pkg/scheduler/scheduler_test.go#L55

Should I rename all imports under package scheduler to `testingframework` ?

Reply:
I see. Let's leave it then